release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2058,6 +2058,87 @@ $ oc annotate -n <NAMESPACE> route/<ROUTE-NAME> "haproxy.router.openshift.io/bal
+
+* Deploying an {product-title} version 4.9 SNO cluster after a successful 4.8 deployment on the same host fails due to a modified boot table entry. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011306[*BZ#2011306*])
+
+* There is an instability issue with the inbox iavf driver that is evident when a DPDK-based workload is deployed in {product-title} version 4.8.5. The issue is also apparent when a DPDK workload is deployed on a host running RHEL for Real Time 8. The issue occurs in hosts with Intel XXV710 NICs installed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2000180[*BZ#2000180*])
+
+* A clock jump error occurs in the `linuxptp` subsystem that is deployed by the PTP Operator. The reported error message is: `clock jumped backward or running slower than expected!`. The error is encountered in a host with an Intel Columbiaville E810 NIC installed in a {product-title} version 4.8 or 4.9 cluster. The error is likely Intel ice driver related, rather than an error in the `linuxptp` subsystem. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013478[*BZ#2013478*])
+
+* Sometimes Operator installation fails during zero touch provisioning (ZTP) installation of a DU node. The `InstallPlan` API reports an error. The reported error message is: `Bundle unpacking failed. Reason: DeadlineExceeded`. The error occurs if the Operator installation job exceeds 600 seconds.
++
+To workaround this issue and re-try the Operator install, run the following `oc` commands for the failed Operator:
++
+. Delete the catalog source:
++
+[source,terminal]
+----
+$ oc -n openshift-marketplace delete catsrc <failed_operator_name>
+----
++
+. Delete the install plan:
++
+[source,terminal]
+----
+$ oc -n <failed_operator_namespace> delete ip <failed_operator_install_plan>
+----
++
+. Delete the subscription and wait for the Operator `CatalogSource` and `Subscription` resources to be re-created by the relevant custom resource policy:
++
+[source,terminal]
+----
+$ oc -n <failed_operator_namespace> delete sub <failed_operator_subscription>
+----
++
+.Expected result
++
+The Operator `InstallPlan` and `ClusterServiceVersion` resources are created and the Operator is installed.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2021456[*BZ#2021456*])
+
+* Sometimes when the ZTP installation process finishes provisioning a DU node, the performance profile configuration is not applied to the node and the `MachineConfigPool` resource becomes stuck in an `Updating` state.
++
+To workaround this issue, perform the following procedure:
++
+. Get the name of the failed DU node:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME     CONFIG                         UPDATED   UPDATING   DEGRADED
+master   rendered-master-90fe2b00c718   False     True       False
```

Comment:
Do we need to comply with [this guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language) for terminal outputs too? It might be worth checking with the peer-review-squad.

Reply:
We can in this instance I think, I've removed the "master" reference.